eldritch/encounters.py

```diff
@@ -252,11 +252,134 @@ def Graveyard6(char):
 def Graveyard7(char):
   return events.Nothing()
 
+def Society1(char):
+  move = events.ForceMovement(char, "Woods")
+  encounter = events.Encounter(char, "Woods", 2)
+  accept = events.Sequence([move, encounter], char)
+  return events.BinaryChoice(char, "Accept Ride?", "Yes", "No", accept, events.Nothing())
+def Society2(char):
+  prereq = events.AttributePrerequisite(char, "dollars", 3, "at least")
+  move = events.ForceMovement(char, "Southside")
+  luck = events.Check(char, "luck", -1)
+  move_dream = events.ForceMovement(char, "Dreamlands1")
+  # TODO: don't hard-code.
+  enc = events.GateEncounter(char, "Dreamlands", {"blue", "green", "red", "yellow"})
+  ret = events.ForceMovement(char, "Society")
+  dreamlands =  events.Sequence([move_dream, enc, ret], char)
+  spell = events.Draw(char, "spells", 1)
+  pay = events.Loss(char, {"dollars": 3})
+  searchstacks = events.PassFail(char, luck, spell, dreamlands)
+  library = events.Sequence([pay, searchstacks], char)
+  choice = events.BinaryChoice(char,"Pay $3 to access the private library", "Yes", "No", library, move)
```

Comment:
BinaryChoice can now accept a prerequisite so you no longer need to use PassFail below. This changes the unit tests slightly in that you must now use resolve_to_choice(). See Diner3 for an example.

Reply:
is that update still in a branch, because I rebased, but I'm not seeing that.

Reply:
hmm, I see it in the base code, so I wonder why my rebase didn't work...

Reply:
Okay I think I've updated this now.  Wow..I'm moving so quickly...